composer.json

```diff
@@ -24,7 +25,7 @@
         "symfony/yaml": "~2.7|~3.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~1|~2",
+        "friendsofphp/php-cs-fixer": "^1|^2.1",
```

Comment:
Ah, this is an error. At first I thought I would not be able to support the old list-style rule configuration, but then I wrote the array helper library (`phptype/array`) to specifically support backwards-compatibility, so this change can be reverted! Nice spot :+1: